src/js/libcs/OpSound.js

```diff
@@ -1,48 +1,533 @@
 //*******************************************************
 // and here are the definitions of the sound operators
 //*******************************************************
+var lines = [0];
+var linesDict = {0: 0};
+
+var getAudioContext = function() {
+    var ac = null;
+    if (!window.AudioContext && !window.webkitAudioContext) {
+        console.warn('Web Audio API not supported in this browser');
+    } else {
+        ac = new(window.AudioContext || window.webkitAudioContext)();
+    }
+    return function() {
+        return ac;
+    };
+}();
```

Comment:
Remove `()` here; do not call this function here! We do not need the to initalize the audioContext whenever CindyJS is loaded. Then also `make alltests` does not throw errors (however, you should consider all the warnings from `make jshint`)